model/fetchAllIngredients.js

```diff
@@ -0,0 +1,19 @@
+const supabase = require("../dbConnection.js");
+
+async function fetchAllIngredients() {
+    try {
+        let { data, error } = await supabase
+            .from('ingredients')
+            .select('id, name:ingredient_name');
```

Comment:
I renamed ingredient_name -> name
                ingredient_category-> category
 in DB, could you please change code to ( 'id, name:name, category')

will use this endpoint for the load config function in ReactJS.